egs/csj/asr1/run.sh

```diff
@@ -151,8 +157,32 @@ if [ ${stage} -le 2 ]; then
          data/${train_dev} ${dict} > ${feat_dt_dir}/data.json
 fi
 
+# It takes a few days. If you just want to end-to-end ASR without LM,
```

Comment:
I think you can remove this comment.
CSJ LM training would be finished quickly.